src/App.js

```diff
@@ -1,9 +1,68 @@
+import CoffeeMatch from "./CoffeeMatch";
+import "./assets/aynu.png";
 import "./App.css";
-
 const App = () => {
+  const matches = [
+    {
+      name: "aynu",
+      image: "./assets/aynu.png",
```

Comment:
Use the imported image like this:

```suggestion
      image: AynuImage,
```